xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/test/java/org/xwiki/web/DocumentTagsTest.java

```diff
@@ -90,7 +90,7 @@ void displayTagsWhenNoEditRightsAndTagPluginAvailableAndNoTags() throws Exceptio
         // - The tag label is displayed
```

Comment:
This is not true anymore, please update this comment.

Reply:
Done with the latest commit.